src/index.ts

```diff
@@ -37,18 +39,22 @@ export interface Bundle {
 export interface ExploreOptions {
   /** Exclude "unmapped" bytes from the output */
   onlyMapped?: boolean;
-  /** Generate html */
-  html?: boolean;
-  /** Generate html ans save to specified file path  */
-  file?: string;
+  /** Output result as a string */
+  output?: {
+    format: OutputFormat;
+    /** Filename to save output to */
+    filename?: string;
+  };
   /** Disable removing prefix shared by all sources */
   noRoot?: boolean;
+  /** Replace "what" by "that" map */
   replaceMap?: ReplaceMap;
 }
 
 export interface ExploreResult {
   bundles: ExploreBundleResult[];
-  html?: string;
+  /** Result as a string - either JSON, TSV or HTML */
```

Comment:
This comment makes me think that there should be a `format: OutputFormat` property of this interface as well.

Reply:
I don't think it is necessary - if user set `ExploreOptions.format = 'html'` it will see `ExploreResult.output = '<html>...'`